### ISSUE TITLE
Release 1.1.2 which contains `ReferenceAssemblies.Net.Net70`

### ISSUE THREAD

**Issue by Youssef1313** [dotnet/roslyn-sdk#1047]
Currently, the latest stable on NuGet doesn't contain `ReferenceAssemblies.Net.Net70`.

Can 1.1.2 be released on NuGet.org?

@arkalyanms @jmarolf @sharwell

**Comment by samblackburn** [comment]
We were able to work around this by inlining the change:
https://github.com/dotnet/roslyn-sdk/blob/ba5f8a6bf7b477b44a43ea943d3e831d6301afb7/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/ReferenceAssemblies.cs#L986-L991

**Comment by Youssef1313** [comment]
@sharwell @arkalyanms @jmarolf Can this be addressed soon? The fix from https://github.com/dotnet/roslyn-sdk/pull/988 is very important and can't be easily worked-around.

I won't be able to add `https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json` for now.

**Comment by sharwell** [comment]
This was being handled by a specific individual as a side project previously. All of our automation only handles getting things to the dotnet-tools feed. That person and the second person who would have handled it in their absence were relocated to another team as part of the recent reorganization, so we are short staffed and specifically hit in the area for this issue. I'm not sure what the plans are for resolving it.

**Comment by dfederm** [comment]
Any update on getting this released to NuGet.org? There are some internal repos which need #1097 to resolve some security alerts.

**Comment by sharwell** [comment]
@dfederm If referencing the dotnet-tools feed is not possible in your project, you should be able to manually reference the updated version of the dependencies to avoid the warnings.

**Comment by drew-cooper** [comment]
Is there any update on this? The version of this package on public NuGet (v1.1.1) is now two major releases of .Net behind in terms of the support for ReferenceAssemblies for analyzer tests.
I'm trying to update a solution which includes analyzers to .Net 8.0 and I'm blocked by this issue.

**Comment by elringus** [comment]
Just in case anyone is looking for a simple workaround until this is fixed:

```csproj
<Project Sdk="Microsoft.NET.Sdk">

    <PropertyGroup>
        <TargetFramework>net8.0</TargetFramework>
        <!-- Add nuget registry containing latest package version. -->
        <RestoreAdditionalProjectSources>
            https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json;
        </RestoreAdditionalProjectSources>
    </PropertyGroup>

    <ItemGroup>
        <PackageReference Include="Microsoft.CodeAnalysis" Version="4.8.0"/>
        <!-- Add package version compatible with .NET 8 -->
        <PackageReference Include="Microsoft.CodeAnalysis.CSharp.SourceGenerators.Testing" Version="1.1.2-beta1.23431.1"/>
        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.8.0"/>
    </ItemGroup>

</Project>
```

**Comment by kdubau** [comment]
@sharwell we need to CG fixes from https://github.com/dotnet/roslyn-sdk/pull/1148 - when will 1.1.2 be release on nuget.org? This is now a critical security hole.

**Comment by b-straub** [comment]
Now .NET8.0 is out quite a long time and still preview is needed to use 'ReferenceAssemblies.Net.Net80'. What is the logic for having such a long preview phase?

**Comment by sharwell** [comment]
@b-straub This was already covered by https://github.com/dotnet/roslyn-sdk/issues/1047#issuecomment-1523360798

**Comment by sharwell** [comment]
@b-straub It has not. 😄

**Comment by MattKotsenas** [comment]
If we're unable or unwilling to push out new versions of the package, should we be recommending it the [cookbook](https://github.com/dotnet/roslyn/blob/main/docs/features/incremental-generators.cookbook.md#unit-testing-of-generators)?

As a generator author, either the package is supported, in which case the [Secure Future Initiative](https://blogs.microsoft.com/on-the-issues/2023/11/02/secure-future-initiative-sfi-cybersecurity-cyberattacks/) suggests our defaults shouldn't include vulnerable packages when there are secure versions available, or it's unsupported and we shouldn't recommend it in the docs.

**Comment by Youssef1313** [comment]
It looks like 1.1.2 is now released already. @sharwell I'm going to close this, assuming there are no issues for releasing subsequent future versions.

**Comment by LordZoltan** [comment]
Since nobody else has - @sharwell, and whoever else needs thanking, THANK YOU!

I'm porting source generators to the incremental way (my own fault they were written as source generators initially, for not going further down the search results... 🤬), which has then meant biting the analyzer bullet (because it is just easier and cleaner).

This package being published has made getting some early examples up and running really easy, and I now know I have a simple reliable base that my programmers will be able work with.  Probably saved us about £30k in dev time over this year.
